Sources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict/>
+	</array>
```

Comment:
Could this be simplified to an empty array like so `<array/>`?

Reply:
Suppose this file is generated from Xcode, maybe I just accidentally press the "add" button to create a new dict, let me have a check.

Reply:
Should be fixed, please review once you are free, thanks a lot @dusi !